_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+
+# A git hook script that runs the same checks that are 
+# executed on Travis to verify your commits will not fail.
+# Bypass it with the --no-verify option to git-commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+# Run 
+./_tests/travis-checks.sh --disable-external
+
+if [ $? -eq 0 ]; then
+    exit 0
+else
+  echo "**********************************************************"
+  echo -e "*  \033[31mCOMMIT ABORTED! Please address the errors before commiting.\033[0m"
```

Comment:
OK, removed error message formatting and added ```err()``` function.  Didn't comment this function as it seemed small an obvious, but if you prefer a comment for this function I can add one.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/81/commits/3f6000223f6819c18a5133a60e77369a825dff00